### DISCUSSION TITLE
LS210d script to use device power switch

### DISCUSSION THREAD

**Discussion by blaudp79** [1000001101000/Debian_on_Buffalo#211] [General]
Hi Everyone,

Based on the post install options wiki, the power switch can be activated on my device after installing triggerhappy and configuring a script.

I have captured the event that that gets registered when the switch is toggled. How do I configure a script to shut down the unit when this event is registered?

Power off switch position event:
EV_KEY  KEY_ESC 0       /dev/input/event0
# KEY_ESC       0       command

Power on switch position event:
EV_KEY  KEY_ESC 1       /dev/input/event0
# KEY_ESC       1       command 

Thanks.

**Comment by 1000001101000** [comment]
See https://github.com/wertarbyte/triggerhappy/blob/master/README
